src/model/operation/transform.js

```diff
@@ -500,10 +806,8 @@ function _getComplementaryAttributeOperations( insertOperation, key, newValue )
 setTransformation( AttributeOperation, MergeOperation, ( a, b ) => {
 	const ranges = [];
 
-	// Case 1:	Attribute change on the merged element. In this case, the merged element was moved to graveyard.
+	// Case 1:	Attribute change on the merged element. In this case, the merged element was moved to the graveyard.
 	//			An additional attribute operation that will change the (re)moved element needs to be generated.
```

Comment:
Wrong indentation, use spaces instead of tabs.

Reply:
It's the same everywhere in the file. I'll change them all.